Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -1482,28 +1497,25 @@ def _attach_algorithm_func_as_method(method_name, algorithm_wrapper, algm_object
 # but we need to do this earlier
 setattr(mantid, MODULE_NAME, sys.modules['mantid.{}'.format(MODULE_NAME)])
 try:
-    _plugins_key = 'python.plugins.directories'
-    _user_key = 'user.%s' % _plugins_key
-    plugin_dirs = _plugin_helper.get_plugin_paths_as_set(_plugins_key)
-    plugin_dirs.update(_plugin_helper.get_plugin_paths_as_set(_user_key))
-    _update_sys_paths(plugin_dirs, recursive=True)
-
-    # Load
-    plugin_files = []
-    alg_files = []
-    for directory in plugin_dirs:
-        try:
-            all_plugins, algs = _plugin_helper.find_plugins(directory)
-            plugin_files.extend(all_plugins)
-            alg_files.extend(algs)
-        except ValueError as exc:
-            logger.warning('Exception encountered during plugin discovery: {0}'.format(str(exc)))
-            continue
+    # Use a cmake generated manifest of all the python algorithms to load them into the api
+    plugins_manifest_path = ConfigService.Instance()["python.plugins.manifest"]
```

Comment:
There's a couple of cases we need to handle here:
- `plugins_manifest_path` is empty to be to deliberately disable loading
- `plugins_manifest_path` is not a valid path so `open` will raise an `OSError`

Reply:
If the manifest is empty should we be launching anyway or fail out with an appropriate error?

Reply:
We can still launch. If a user wants to disable the python algorithms then as long as they don't try to call one it should be fine.

Reply:
It would be the same as blanking the `python.plugins.directories` currently.